app/src/ui/index.tsx

```diff
@@ -1,3 +1,4 @@
+import 'react-devtools'
```

Comment:
Can we hide this behind the `__DEV__` global? The plan is for the underlying package to be removed when building for release, right?